cli/src/main/java/io/micronaut/guides/TweetsCommand.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2017-2021 original authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.micronaut.guides;
+
+import io.micronaut.configuration.picocli.PicocliRunner;
+import io.micronaut.guides.httpclient.Guide;
+import io.micronaut.guides.httpclient.GuidesClient;
+import jakarta.inject.Inject;
+import picocli.CommandLine.Command;
+import picocli.CommandLine.Option;
+import reactor.core.publisher.Mono;
+import java.util.ArrayList;
+import java.util.List;
+
+@Command(name = "guidetweets", description = "generates a list of tweets", mixinStandardHelpOptions = true)
+public class TweetsCommand implements Runnable {
+    private static final String HASHTAG = " #";
```

Comment:
Why do you need the extra space before the `#`. There is already one space in the generated text:

```
📖 Micronaut Guide: Micronaut JWT Authentication https://guides.micronaut.io/latest/micronaut-security-jwt.html  #micronaut
```